src/app/core/services/tracking.service.ts

```
@@ -158,24 +158,24 @@
   // external APIs
-  onSignin(email: string, properties: { label?: string } = {}): void {
+  onSignin(userId: string, properties: { label?: string } = {}): void {
     try {
-      mixpanel?.identify(email);
+      mixpanel?.identify(userId);
     } catch (e) {}
 
-    this.identityEmail = email;
+    this.identityId = userId;
     this.eventTrack('Signin', properties);
   }
```

ESLint

[error] Member onSignin should be declared before all private instance method definitions.

(@typescript-eslint/member-ordering)